From c6e66393ba5ed91b02f0bfbf7367771ce7ba819c Mon Sep 17 00:00:00 2001
From: Simon Howard <fraggle@soulsphere.org>
Date: Sun, 12 Jul 2015 02:52:57 -0400
Subject: [PATCH 06/13] osx: Add CFBundleIdentifier to Info.plist.

The preferences for the OS X launcher are currently stored in a file
ambiguously named "launcher.plist" that could conflict with any other
app on the system that chose to use the same moniker. Use the more
specific identifier of org.chocolate-doom.launcher to avoid conflicts.

Also set NSHumanReadableCopyright so that we get more detail on the
"About" window in the launcher app.

This fixes #570. Thanks to Xeriphas1994 for the suggestion to tweak
the identifier.
---
 pkg/osx/Info.plist.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/pkg/osx/Info.plist.in b/pkg/osx/Info.plist.in
index 5ea4f32..a648ed8 100644
--- a/pkg/osx/Info.plist.in
+++ b/pkg/osx/Info.plist.in
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+        <key>CFBundleIdentifier</key>
+        <string>org.chocolate-doom.launcher</string>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleDisplayName</key>
@@ -26,6 +28,10 @@
 	<string>NSApplication</string>
 	<key>NSMainNibFile</key>
 	<string>launcher</string>
+        <key>NSHumanReadableCopyright</key>
+        <string>Copyright (C) 1993-2015, id Software and Raven Software, Simon Howard, James Haley, Samuel Villarreal and other contributors.
+
+Licensed under the GNU GPL v2.</string>
 
         <!-- file associations: -->
 
-- 
2.5.0

